projects/plugins/jetpack/_inc/client/my-plan/my-plan-header/index.js

```diff
@@ -343,6 +343,26 @@ class MyPlanHeader extends React.Component {
 					title: __( 'Jetpack VideoPress', 'jetpack' ),
 				};
 
+			case 'is-jetpack-social-basic-plan':
+				return {
+					...productProps,
+					details: [ activation, expiration ],
+					tagLine: __( 'Share your content to social media sites.', 'jetpack' ),
```

Comment:
The other tag lines seem to be of the form 'You can...', 'Your site is...' - focusing an the features 'you' are benefiting from. We should probable rephrase that here, but I'm not sure what's best. "You can automatically share your content to social media sites.", maybe?